example_user.rb

```diff
@@ -1,5 +1,6 @@
 class User
   attr_accessor :name, :email
+
```

Comment:
Extra changes (don't fix)
But do not add changes that do not apply to the current task in the future.